wafer/pages/models.py

```diff
@@ -62,9 +68,26 @@ def get_in_schedule(self):
             return True
         return False
 
+    def get_people_display_names(self):
+        names = []
+        for person in self.people.all():
+            full_name = person.get_full_name()
+            if full_name:
+                names.append(full_name)
+            else:
+                names.append(person.username)
+        if len(names) > 2:
+            # At some future point, I'll do something better
```

Comment:
Maybe we should just remove this comment? I'm not sure what we can do that is better.